kahuna/public/js/components/gr-image-usage/gr-image-usage.js

```diff
@@ -5,25 +5,31 @@ import template from './gr-image-usage.html!text';
 import usageTemplate from './gr-image-usage-list.html!text';
 import './gr-image-usage.css!';
 
-export let module = angular.module('gr.imageUsage', []);
+import '../../image/service';
 
-module.controller('grImageUsageCtrl', ['mediaUsage', function (mediaUsage) {
-    const ctrl = this;
+export let module = angular.module('gr.imageUsage', [
```

Comment:
We really should almost never have to use `let`. `const` means constant reference, which almost all our code (including this) does, *not* immutable value.